webapp/app/forms/steps/lotse/personal_data.py

```diff
@@ -523,4 +523,136 @@ def render(self):
                                props=props_dict,
                                form=self.render_info.form,
                                header_title=_('form.lotse.header-title'))
-        
\ No newline at end of file
+
+class PersonAHasDisabilityPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepPersonAHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+
+    @root_validator(skip_on_failure=True)
+    def person_b_has_disability(cls, values):
+        if values.get('person_a_has_disability') != 'yes':
+            raise ValidationError
+        return values
+        
+class PersonBHasDisabilityPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepPersonBHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+
+    @root_validator(skip_on_failure=True)
+    def person_b_has_disability(cls, values):
+        if values.get('person_b_has_disability') != 'yes':
+            raise ValidationError
+        return values
+    
+class StepPauschbetragPersonA(LotseFormSteuerlotseStep):
+    name = 'person_a_requests_pauschbetrag'
+    section_link = SectionLink('mandatory_data', StepFamilienstand.name, _l('form.lotse.mandatory_data.label'))
+
+    preconditions = [PersonAHasDisabilityPrecondition]
+    
+    class InputForm(SteuerlotseBaseForm):
+        person_a_requests_pauschbetrag = SelectField(
+            # TODO: This is because of key mapping - the text is within the react component
```

Comment:
I think I do not understand the TODO comment? Could you maybe explain what it means?

Reply:
In my opinion, we don't need the content of the key in the backend. In the backend I would just like to define my keys but the text should be defined in the react component. Also, here the content does not match the frontend and can lead to confusion why empty texts are written in the backend.

Reply:
Maybe we could rewrite the todo comment: 

```suggestion
            # TODO: The empty strings are because of key mapping - the text is within the react component
```